.vscode/extensions.json

```diff
@@ -3,6 +3,7 @@
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
     "redhat.vscode-yaml",
-    "streetsidesoftware.code-spell-checker"
+    "streetsidesoftware.code-spell-checker",
+    "orta.vscode-jest"
```

Comment:
The only Jest extension that supports multi-root workspaces AFAIK, and each of them has very bad monorepo support.